### PR TITLE
Rearrange logo options

### PR DESCRIPTION
**Pull request by classicrocker883**
### Description
adds the option to choose a small static Marlin logo, large animated, or both enabled = a small animated logo
(as per src/lcd/dogm/dogm_Bootscreen.h)

added a line between two `env:` in stm32f4.ini

### Benefits
helps gives user the options and clarifies what it does.

wording and layout may be considered for change

### CODE REVIEW

File: Marlin/Configuration_adv.h [hunk 1]
```diff
@@ -1531,8 +1531,9 @@
       #define BOOTSCREEN_TIMEOUT 4000       // (ms) Total Duration to display the boot screen(s)
       #if EITHER(HAS_MARLINUI_U8GLIB, TFT_COLOR_UI)
         #define BOOT_MARLIN_LOGO_SMALL      // Show a smaller Marlin logo on the Boot Screen (saving lots of flash)
+        //#define BOOT_MARLIN_LOGO_ANIMATED   // Choose either small static logo, large animated logo, or BOTH shows small animated logo
```
Comment (thisiskeithb) [inline Marlin/Configuration_adv.h]: `TFT_COLOR_UI` does not have an animated boot screen, so `BOOT_MARLIN_LOGO_ANIMATED` does not apply.

File: Marlin/Configuration_adv.h [hunk 2]
```diff
@@ -1531,8 +1531,9 @@
...
       #endif
-      //#define SHOW_CUSTOM_BOOTSCREEN      // Show the bitmap in Marlin/_Bootscreen.h on startup.
+      #define SHOW_CUSTOM_BOOTSCREEN      // Show the bitmap in Marlin/_Bootscreen.h on startup.
```
Comment (thisiskeithb) [inline Marlin/Configuration_adv.h]: This should be left disabled by default.

File: Marlin/Configuration_adv.h (hunk 1 above)
Comment (classicrocker883) [reply Marlin/Configuration_adv.h]: well I tried. at least the cleanup for stm32f4.ini is something

**Review by classicrocker883 — COMMENTED** [review]
```
  #if ANY(HAS_MARLINUI_U8GLIB, TOUCH_UI_FTDI_EVE, IS_DWIN_MARLINUI)
    #define SHOW_BOOTSCREEN                 // Show the Marlin bootscreen on startup. ** ENABLE FOR PRODUCTION **
    #if ENABLED(SHOW_BOOTSCREEN)
      //#define SHOW_CUSTOM_BOOTSCREEN      // Show the bitmap in Marlin/_Bootscreen.h on startup.
      #define BOOTSCREEN_TIMEOUT 4000       // (ms) Total Duration to display the boot screen(s)
      #if EITHER(HAS_MARLINUI_U8GLIB, TFT_COLOR_UI)
        #define BOOT_MARLIN_LOGO_SMALL      // Show a smaller Marlin logo on the Boot Screen (saving lots of flash)
      #endif
    #endif
```

would moving `SHOW_CUSTOM_BOOTSCREEN` be better moved up like so, or no?

**Pull request merged**